src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -3,6 +3,10 @@
 
 #include "common.h"
 
+#ifdef FEATURE_NATIVEAOT
+#include "RhConfig.h"
```

Comment:
I do not think we need this for native AOT. We have been staying away from the runtime config like this for native AOT.

Reply:
I was not sure if NativeAOT must match CoreCLR here or not. 
I do not expect a lot of uses for this knob. It is just something that we can have.

Reply:
Yes, I expect that we will delete this after a while.